backend/entityservice/encoding_storage.py

```diff
@@ -110,3 +121,54 @@ def get_encoding_chunk(conn, chunk_info, encoding_size=128):
     chunk_data = binary_unpack_filters(encoding_iter, encoding_size=encoding_size)
     return chunk_data, len(chunk_data)
 
+
+def upload_clk_data_binary(project_id, dp_id, encoding_iter, receipt_token, count, size=128):
+    """
+    Save the user provided binary-packed CLK data.
+
+    """
+    filename = None
+    # Set the state to 'pending' in the uploads table
+    with DBConn() as conn:
+        db.insert_encoding_metadata(conn, filename, dp_id, receipt_token, encoding_count=count, block_count=1)
```

Comment:
what's with the `filename = None` business?

Reply:
It is just a way of keeping the column in the `uploads` table set to NULL.